ixmp/default_paths.py

```diff
@@ -12,22 +17,32 @@ def db_config_path():
 
 
 def find_dbprops(fname):
-    """Search directories for file fname. First start in local dir (`.`), then look
-    in ixmp default locations.
+    """Return the absolute path to a database properties file.
+
+    Searches for a file named *fname*, first in the current working directory
+    (`.`), then in the ixmp default location.
 
     Parameters
     ----------
-    fname : string
-        filename
+    fname : str
+        Name of a database properties file to locate.
+
+    Returns
+    -------
+    str
+        Absolute path to *fname*.
+
+    Raises
+    ------
+    FileNotFoundError
+        *fname* is not found in any of the search paths.
     """
-    # look local first
-    if os.path.isfile(fname):
-        return fname
-
-    # otherwise look in default directory
-    config_path = db_config_path()
-    _fname = os.path.join(config_path, fname)
-    if not os.path.isfile(_fname):
-        raise IOError('Could not find {} either locally or in {}'.format(
-            fname, config_path))
-    return _fname
+    # Look in the current directory first, then the configured directory
+    dirs = ['', db_config_path()]
+
+    for dir in dirs:
```

Comment:
`dir` is a python keyword, so some other variable name would be better here

Reply:
Renamed `directory`